pkg/registry/servicecatalog/clusterserviceplan/storage.go

```diff
@@ -142,8 +144,30 @@ func NewStorage(opts server.Options) (rest.Storage, rest.Storage) {
 		CreateStrategy: clusterServicePlanRESTStrategies,
 		UpdateStrategy: clusterServicePlanRESTStrategies,
 		DeleteStrategy: clusterServicePlanRESTStrategies,
-		Storage:        storageInterface,
-		DestroyFunc:    dFunc,
+
+		TableConvertor: tableconvertor.NewTableConvertor(
+			[]metav1beta1.TableColumnDefinition{
+				{Name: "Name", Type: "string", Format: "name"},
+				{Name: "External name", Type: "string"},
+				{Name: "Broker", Type: "string"},
```

Comment:
This should be ClusterServiceBrokerName

Reply:
This will be the printed name, correct? I thought the point of tweaking the default printing was to get closer to what svcat does (and a huge part of that is making our field names shorter and more intuitive). What's wrong with just `Broker`?

Reply:
I don't see the value of calling the column `CLUSTER-SERVICE-BROKER-NAME`, as it just makes the output unnecessarily wide:

```
NAME                                   EXTERNAL-NAME                        CLUSTER-SERVICE-BROKER-NAME   BINDABLE   PLAN-UPDATABLE   AGE
4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468   user-provided-service                ups-broker                    true       true             12m
5f6e6cf6-ffdd-425f-a2c7-3c9258ad2468   user-provided-service-single-plan    ups-broker                    true       true             12m
8a6229d4-239e-4790-ba1f-8367004d0473   user-provided-service-with-schemas   ups-broker                    true       true             12m
```